clouddriver-google/src/main/groovy/com/netflix/spinnaker/clouddriver/google/deploy/validators/ModifyGoogleServerGroupInstanceTemplateDescriptionValidator.groovy

```diff
@@ -45,5 +45,8 @@ class ModifyGoogleServerGroupInstanceTemplateDescriptionValidator extends Descri
     helper.validateAuthScopes(description.authScopes)
     helper.validateRegion(description.region)
     helper.validateServerGroupName(description.serverGroupName)
+    if (description.instanceType) {
+      helper.validateInstanceType(description.instanceType, description.zone)
```

Comment:
This should actually be `description.region`, since `ZoneToRegionDescriptionPreProcessor` drops `zone` in favor of `region`.